src/include/sof/audio/volume.h

```diff
@@ -75,12 +75,12 @@ struct sof_ipc_ctrl_value_chan;
 /**
  * \brief Macro for volume linear gain ramp step computation
  * Volume gain ramp step as Q1.16 is computed with equation
- * step = VOL_RAMP_STEP_CONST/ SOF_TKN_VOLUME_RAMP_STEP_MS. This
+ * step = VOL_RAMP_STEP_CONST / SOF_TKN_VOLUME_RAMP_STEP_MS. This
  * macro defines as Q1.16 value the constant term
- * (1000 / VOL_RAMP_UPDATE) for step calculation.
+ * VOL_RAMP_UPDATE / 1000.0 for step calculation.
```

Comment:
Where does 1000 come from ?

Reply:
It's to scale between topology defined ms and scheduler us units. I'll add comments.

Reply:
I forgot this, PR update coming...